builder/builder-next/adapters/containerimage/pull.go

```diff
@@ -21,7 +21,6 @@ import (
 	"github.com/containerd/containerd/v2/core/remotes/docker"
 	"github.com/containerd/containerd/v2/core/remotes/docker/schema1" //nolint:staticcheck // Ignore SA1019: "github.com/containerd/containerd/remotes/docker/schema1" is deprecated: use images formatted in Docker Image Manifest v2, Schema 2, or OCI Image Spec v1.
 	"github.com/containerd/containerd/v2/pkg/gc"
-	cdreference "github.com/containerd/containerd/v2/pkg/reference"
 	ctdreference "github.com/containerd/containerd/v2/pkg/reference"
```

Comment:
As we're updating (and looks like this is the only place we use an alias), perhaps we should use `c8dreference` (we use `c8d` as short prefix for containerd imports in various places.